### ISSUE TITLE
Accessibility: Missing focus styles for keyboard navigation

### ISSUE THREAD

**Issue by glombek** [PerplexDigital/Perplex.ContentBlocks#4]
A quick fix may be to add the `:focus` pseudoclass to any location where you already have `:hover`.

Happy to contribute here, but thought I'd make an issue initially.

**Comment by PerplexDaniel** [comment]
Hey glombek, we will be happy to look at a PR. There will probably be more issues regarding keyboard navigation though, I am not aware of much attention to that area during development and personally am not testing that either so no idea what the current state is.

**Comment by glombek** [comment]
You're right! It's going to be a big one!

I'm going to try and make a first pass to make it possible to navigate the main journey.

I've made a (very small) start https://github.com/PerplexDigital/Perplex.ContentBlocks/compare/master...glombek:feature/keyboard-navigation?expand=1

But this will then need:
- All UX to work
- Clearer focus styles (that aren't just the `:hover` styles) so selection is more obvious for keyboard users.

**Comment by glombek** [comment]
Would also be good to repurpose https://github.com/umbraco/Umbraco-CMS/pull/8141 for modals. So this pending acceptance into core.

**Comment by PerplexDaniel** [comment]
Hey @glombek,

If those `:focus` additions to the CSS do not break anything and already make it (slightly) better to navigate with keyboard feel free to make a PR for that alone and I'll happily merge it. Small improvements are also improvements.

Regarding that modal, we are unlikely to use new Umbraco directives that are only available in very new versions because our package is compatible with Umbraco 8.1+. It is possible only if including that new directive does not generate any errors in older installations and yields a better UX in newer versions. Anyway once that is in core we can discuss that in another issue :)

**Comment by glombek** [comment]
They do very little currently, so will need to spend a little more time on them (some require `:focus-within` etc first) but will PR as soon as I have something!

Sorry, i was unclear about the modal. I'm thinking that, so long as ContentBlocks uses the modal directive, we can ignore that issue until it's automatically fixed by core.

**Comment by PerplexDaniel** [comment]
Closed due to 4 weeks of inactivity.
Feel free to reopen with new input.

**Comment by glombek** [comment]
This issue should not be closed due to inactivity - these will take time to fix but remain severe accessibility issues.
